source/clear-linux/guides/maintenance/kernel-development.rst

```diff
@@ -6,26 +6,28 @@ Kernel development
 This document shows how to obtain and compile a Linux* kernel source
 using |CL-ATTR| development tooling.
 
-The `kernels available`_  in |CL| aim to be performant and practical. In some
-cases, it may be necessary to modify the kernel to suit your specific needs
-or test new kernel code as a developer.
+The :ref:`compatible-kernels` available in |CL| aim to be performant and
+practical. In some cases, it may be necessary to modify the kernel to suit your
+specific needs or test new kernel code as a developer.
 
 .. contents::
    :local:
    :depth: 1
    :backlinks: top
 
-Source RPM files (SRPM) are also available for all |CL| kernels, and can be
-used for development instead. Select this link to view the latest `source RPM files`_.
+Source RPM (SRPM) files are also available for all |CL| kernels, and can be
+used for development instead. Select this link to view the latest
+`SRPM`_ files.
```

Comment:
[Index page](https://cdn.download.clearlinux.org/current/source/SRPMS/) uses "SRPMS", please add S to match it.

Reply:
Good catch - thanks :)